doc/source/train/user-guides/persistent-storage.rst

```diff
@@ -29,7 +29,7 @@ Here are some capabilities that persistent storage enables:
   and artifacts to share them with others or use them in downstream tasks.
 
 
-Cloud storage (AWS S3, Google Cloud Storage)
+Cloud storage (AWS S3, Google Cloud Storage, Azure Blob)
```

Comment:
Have we tested this? I'm actually not sure whether azure blob is supported or not here.